index.html

```diff
@@ -88,9 +90,9 @@
 <h2>
 <a name="installing-sparkr" class="anchor" href="#installing-sparkr"><span class="octicon octicon-link"></span></a>Installing SparkR</h2>
 
-<p>SparkR requires Scala 2.10 and Spark version &gt;= 1.1.0 (support for Spark 1.3.0 coming soon) and depends on R packages <code>testthat</code> (only required for running unit tests). </p>
+<p>SparkR requires Scala 2.10 and Spark version &gt;= 1.1.0 and depends on the R package <code>testthat</code> (only required for running unit tests).</p>
```

Comment:
Can you add a note about the master vs. sparkr-sql branch information here as well ?